contracts/src/NobleDollar.sol

```diff
@@ -142,7 +146,7 @@ contract NobleDollar is HypERC20 {
             revert NoClaimableYield();
         }
 
-        _transfer(address(this), msg.sender, amount);
+        _transfer(address(this), msg.sender, amount); // Consider update() instead?
```

Comment:
`_transfer` is identical to `_update` with the added checks that the sender/recipient aren't the zero-address. I'm happy to change this 😄


```suggestion
        _update(address(this), msg.sender, amount);
```

Reply:
I am for _update to avoid unnecessary revert checks at this stage